packages/block-library/src/post-tags/edit.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+
+function PostTagsDisplay() {
+	const [ tags ] = useEntityProp( 'postType', 'post', 'tags' );
+	const tagLinks = useSelect(
+		( select ) => {
+			const { getEntityRecord } = select( 'core' );
+			let loaded = true;
+			const links = tags.map( ( tagId ) => {
+				const tag = getEntityRecord( 'taxonomy', 'post_tag', tagId );
+				if ( ! tag ) {
+					return ( loaded = false );
+				}
+				return (
+					<a key={ tagId } href={ tag.link }>
+						{ tag.name }
+					</a>
+				);
```

Comment:
I believe ultimately, we could offer a way to add/remove tags directly here. Let's land a readonly block first though.

Reply:
Yes